ceph_salt/execute.py

```diff
@@ -1390,6 +1390,40 @@ def ping_minions():
                 retval = 8
         return retval
 
+    @staticmethod
+    def check_dns():
+        retval = None
+        PP.println("Checking if all minions have functioning DNS...")
+        minion_hostnames = CephNodeManager.list_all_minions()
```

Comment:
This will check all minions, but I think we only care about the ones that are part of `ceph-salt` configuration (pillar), right?

Reply:
fixed